### ISSUE TITLE
Suddenly got "Chrome is not reachable" in Selenium - Lambda JAVA17 Integration

### ISSUE THREAD

**Issue by ZehavitSaban** [aws/aws-sam-build-images#145]
### Description:
It looks like Amazon released a new java17 image after March 19th that caused this issue:
"Chrome is not reachable"

### How to reproduce:
SE VERSION -  4.21.0
CHROME VERSION - 103.0.5058.0 (LAYER)
First, we build and deploy lambda with the following dependencies. So, when we invoke the Lambda, it already comes with these dependencies pre-installed on the Lambda.

```
dependencies {
    implementation (
            'com.amazonaws:aws-java-sdk-s3:1.12.730',
            'com.amazonaws:aws-lambda-java-core:1.2.3',
            'org.apache.commons:commons-lang3:3.14.0',
            'commons-io:commons-io:2.15.1',
            'org.apache.logging.log4j:log4j-api:2.22.0',
            'org.apache.logging.log4j:log4j-core:2.22.0'
    )
}

```

Inside the lambda, we use S3 to get the company's internal automation framework and the test code we want to run:

```
echo "running java (time left: $timeLeft)"
timeout ${timeLeft}s java -Dwebdriver.chrome.driver=$dir/drivers/chromedriver -Dchrome.binary=$dir/headless-chromium 
 -cp $jar com.duda.webdriver.elduderino.core.Main $test $url $retry $params 2>&1
```

Basically, before 19.3 it worked fine. We could run tests inside AWS Lambda and then integrate it with our CI/CD.
But, after your changes we can't.

Workaround:
I changed from the latest version to a previous released version (which was before 19.3) and it seems to solve the problem.
I would like to know what change caused this problem and how I should fix my code to support this change.

**Comment by jysheng123** [comment]
Hi, thanks for bringing up the issue. Would you be able to provide a sample lambda function that has this error occurring? It would help the debugging process a lot in assisting us to create steps to reproduce the issue

**Comment by ZehavitSaban** [comment]
Hi @jysheng123 , thanks for the support!
I created  a sample lambda function. In our native lambda we use chromeLayer with chrome version 103. Our runtime is Java 17.

[SeleniumLambda.zip](https://github.com/user-attachments/files/15594681/SeleniumLambda.zip)

**Comment by ZehavitSaban** [comment]
Hi @jysheng123 , can you please assist? We still have this issue.

**Comment by ZehavitSaban** [comment]
Hi @jysheng123 , can you please assist? We still have this issue.

**Comment by seshubaws** [comment]
Hello @ZehavitSaban, sorry for the delay in response. Please confirm if you are still having this issue with the latest Java17 image. I will try to reproduce this issue with the latest Java17.

Note that the version of the Chrome browser and the version of chromedriver must match the major version.

Additionally, if you are using this with a SAM template, it would be helpful if you could provide that as well.
